python/tvm/intrin.py

```diff
@@ -285,6 +285,22 @@ def trunc(x):
     return call_pure_intrin(x.dtype, "trunc", x)
 
 
+def absolute(x):
+    """Get absolute value of the input element-wise.
+
+    Parameters
+    ----------
+    x : Expr
+        Input argument.
+
+    Returns
+    -------
+    y : Expr
+        The result.
+    """
+    return call_pure_intrin(x.dtype, "fabs", x)
```

Comment:
Please also add support dispatching integer values, implement this function in the c++ side.

```c++
Expr tvm::abs(Expr x) {
   if (x.type().is_int()) {
      return select(x >= make_zero(x.type()), x, -x);
   } else if (x.type().is_float()) {
     return call_pure_intrin("fabs")...
   } else if (x.type().is_uint()) {
      return x;
   }
}
```

Reply:
implementation should go to here https://github.com/dmlc/tvm/blob/master/include/tvm/ir_operator.h#L61